src/widgets/transformer-editor.jsx

```diff
@@ -659,15 +659,15 @@ var ShapeTypes = {
                 return false;
             }
 
-            var line1_0 = [points1[1], points1[0]];
-            var line1_2 = [points1[1], points1[2]];
-            var line2_0 = [points2[1], points2[0]];
-            var line2_2 = [points2[1], points2[2]];
+            var line1a = [points1[1], points1[0]];
```

Comment:
Hmm, kinda silly that it yells about this. It's hard to capitalize a zero! I'd vote for just disabling the lint rule, since the original syntax made more sense.